apps/mdn/mdn-aws/infra/acm.tf

```diff
@@ -29,6 +29,11 @@ data "aws_acm_certificate" "prod-media-cdn-cert" {
   domain   = "media.prod.mdn.mozit.cloud"
 }
 
+data "aws_acm_certificate" "interactive_examples_stage_mdn_mozit_cloud" {
+  provider = aws.acm
+  domain   = "interactive-examples.stage.mdn.mozilla.net"
```

Comment:
This doesn't seem to be a public domain? Could we make it on the mozit.cloud or mozilla/allizom domain?